packages/oui-navbar/_mixins.less

```diff
@@ -268,6 +168,157 @@
     }
   }
 
+  .navbar-menu-animation_slide(
+    @origin: left
+  ) {
+    opacity: 1;
+    visibility: hidden; // For ARIA
+    pointer-events: auto;
+    transition: transform 0.25s ease-out, visibility 0.25s ease-out;
+
+    & when (@origin = left) {
+      transform: translateX(-100%);
+    }
+
+    & when (@origin = right) {
+      transform: translateX(100%);
+    }
+
+    [aria-expanded="true"] ~ & {
+      visibility: visible;
+      transform: translateX(0);
+      transition: transform 0.25s ease-out;
+    }
+  }
+
```

Comment:
Is it voluntary to have so many empty lines?

Reply:
Yes, it's to mark the separation for the deprecated mixins